contrib/terraform/terraform.py

```diff
@@ -21,12 +21,12 @@
 directory and generates an inventory based on them.
 """
 import argparse
-from collections import defaultdict
-import random
-from functools import wraps
 import json
 import os
+import random
 import re
+from collections import defaultdict
+from functools import wraps
```

Comment:
Why do we need to change the order of `import`s here?

Reply:
ah its because i have isort enabled for all my python work:
- https://pypi.org/project/isort/

Reply:
That seems unrelated to the purpose of this pull request, right?
If so, please have another separated pull request to keep the pull request small and simple.

Reply:
okay thanks no problem